src/Itinero/Algorithms/Shortcuts/RouterDbExtensions.cs

```diff
@@ -30,7 +31,7 @@ public static class RouterDbExtensions
         /// <summary>
         /// Adds multiple shortcut db's at the same time.
         /// </summary>
-        public static void AddShortcuts(this RouterDb routerDb, params ShortcutSpecs[] shortcutSpecs)
+        public static void AddShortcuts(this RouterDb routerDb, CancellationToken cancellationToken = new CancellationToken(), params ShortcutSpecs[] shortcutSpecs)
```

Comment:
I'm not in love with how this makes the callers look... maybe this is one of those cases where it makes sense to add an overload with the cancellation token at the cost of the cancellation-aware overload's `shortcutSpecs` not being `params`?